packages/iTwinUI-react/src/core/Table/TableRowMemoized.tsx

```diff
@@ -74,18 +74,19 @@ export const TableRow = <T extends Record<string, unknown>>(props: {
     root: intersectionRoot,
   });
 
-  const userRowProps = rowProps?.(row);
+  const userRowProps = rowProps?.(row) ?? {};
+  const { status, ...restUserRowProps } = userRowProps;
```

Comment:
To confirm, is `...restUserRowProps` equal to all the attributes in `userRowProps` other than `status`?

Reply:
Yes.
We want to exclude `status` to not show it in DOM as (eg)`status='positive'`. Therefore, we are spreading userRowProps into status and restUserRowProps.